_docs/MAIN.md

```diff
@@ -1,3 +1,16 @@
 # terraform-aws-tardigrade-cloudwatch-log-metric-filter
 
 Terraform module to create CloudWatch Log Metric Filters
+
+# variable definitions
+
+As can be seen in the inputs, this module takes the variable `metric_filters`, which is a schema list consisting of the following fields:
+* name (string) - the name for the metric filter
```

Comment:
This renders a little more clearly, I think, if the variable names are wrapped in backticks...
```suggestion
* `name` (string) - the name for the metric filter
```